packages/legacy/core/App/screens/OnboardingPages.tsx

```diff
@@ -129,16 +135,47 @@ const guides: Array<{ image: React.FC<SvgProps>; title: string; body: string }>
   },
 ]
 
-const createPageWith = (image: React.FC<SvgProps>, title: string, body: string, OnboardingTheme: any) => {
+const createPageWith = (
+  image: React.FC<SvgProps>,
+  title: string,
+  body: string,
+  OnboardingTheme: any,
+  devModeListener?: boolean
+) => {
   const styles = createStyles(OnboardingTheme)
   const imageDisplayOptions = createImageDisplayOptions(OnboardingTheme)
+  const navigation = useNavigation<StackNavigationProp<OnboardingStackParams>>()
+  const [, dispatch] = useStore()
+  const developerOptionCount = useRef(0)
+  const touchCountToEnableBiometrics = 9
+
+  const incrementDeveloperMenuCounter = () => {
+    if (developerOptionCount.current >= touchCountToEnableBiometrics) {
+      developerOptionCount.current = 0
+      dispatch({
+        type: DispatchAction.ENABLE_DEVELOPER_MODE,
+        payload: [true],
+      })
+      navigation.navigate(Screens.Developer)
```

Comment:
Were we not going to navigate to the settings screen? So that we can change language settings etc. Just wondering

Reply:
I think the requirement was just the developer screen so that things like wallet name could be enabled